wordcount/src/sort/Main.java

```diff
@@ -0,0 +1,77 @@
+        String newString = rawString.replaceAll("[[\\.\\?\\!\\,\\;\\:\\{\\}\\(\\)\\']]", ""); //this is the parsed string
+         
+        String[] words = newString.split(" +"); //split and store into words[]
+
```

Comment:
Effective use of the provided Java String methods here. Note the slightly different regex syntax taken in by these methods - the regex is encased in `" "` instead of `/ /` as it would be in Javascript, and the `g` (global match) modifier isn't needed to get multiple matches for the `.replaceAll` method.